src/jobs/deploy_to_selfhosted.yml

```diff
@@ -88,7 +88,7 @@ steps:
         --set global.metrics.grafana.enabled=true \
         --set global.listener.nfsserverprovisioner.storageClass.name="codacy-<<parameters.environment>>-listener-cache-class" \
         --set global.features.cloneSubmodules=true \
-        --set global.worker-manager.config.codacy.worker.image=dev \
+        --set global.workerManager.workers.config.imageVersion=dev \
```

Comment:
We need this here otherwise any deployment from any component will crush this value back to the `stable` version of the worker image